README.md

```diff
@@ -16,27 +16,46 @@ To install with [**Foundry**](https://github.com/gakonst/foundry):
 forge install exp-table/pigeon
 ```
 
+Set environment variables:
+
+```sh
+ETH_MAINNET_RPC_URL=
+POLYGON_MAINNET_RPC_URL=
+
+# Optional
+ENABLE_ESTIMATES=
+```
+
+To enable gas estimation, from the [utils/scripts directory](./utils/scripts), run the following command:
+
+```
+npm install
+npm run compile
+```
+
 ## Usage
 
 It is made to be as simple as instantiating the helper in your test file, and simple calling `help` with the appropriate parameters.
 We have provided examples in the test files.
 
-In short, it looks like this (for Hyperlane) :
+In short, it looks like this (for Hyperlane):
 
 ```js
 vm.recordLogs();
 _someCrossChainFunctionInYourContract(L2_DOMAIN, TypeCasts.addressToBytes32(address(target)));
 Vm.Log[] memory logs = vm.getRecordedLogs();
-hyperlaneHelper.help(0x35231d4c2D8B8ADcB5617A638A0c4548684c7C70, L2_FORK_ID, logs);
+hyperlaneHelper.help(L1_DOMAIN, 0x35231d4c2D8B8ADcB5617A638A0c4548684c7C70, L2_FORK_ID, logs);
```

Comment:
The `L1_DOMAIN` and mailbox address order is inversed. The prototype of the function is `help(address mailbox, uint32 originDomain, ...)`.